.github/workflows/publish.yaml

```
@@ -21,24 +21,24 @@ jobs:
     needs:
       - test
     steps:
-    - name: Checkout repository and submodules
-      uses: actions/checkout@v4
-      with:
-        token: ${{ secrets.SGILE_PAT }}
-        submodules: recursive
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: "3.10"
-    - name: Install pypa/build
-      run: python3 -m pip install build
-    - name: Build a binary wheel and a source tarball
-      run: python -m build --sdist --wheel
-    - name: Store the distribution packages
-      uses: actions/upload-artifact@v4
-      with:
-        name: python-package-distributions
-        path: dist/
+      - name: Checkout repository and submodules
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.SGILE_PAT }}
+          submodules: recursive
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+      - name: Install pypa/build
+        run: python3 -m pip install build
+      - name: Build a binary wheel and a source tarball
+        run: python -m build --sdist --wheel
```

Do we want to change to `hatch` or should we wait?
```suggestion
      - name: Install pypa/build
        run: python3 -m pip install hatch
      - name: Build a binary wheel and a source tarball
        run: hatch build
```